pyFAI/ext/inpainting.pyx

```diff
@@ -44,6 +44,9 @@ from libc.stdint cimport int8_t, uint8_t, int16_t, uint16_t, \
 
 include "bilinear.pxi"
 
+cdef inline double pow2(double x) noexcept nogil:
+    return x*x
+
```

Comment:
Maybe reuse the one from `regrid_common.pxi` or make a dedicated `pxi` file for those, e.g., `math.pxi` or `utils.pxi`

Reply:
I did chose this solution ...